### ISSUE TITLE
Support arguments to remove the need of prompts

### ISSUE THREAD

**Issue by thwonghin** [divineunited/gitutils-cli#2]
Instead of entering the commit / branch name one by one through prompts, can it support inputting all of them at once with arguments to streamline it?

For example
```
gitutils-cherry-pick --commit aa262461 --to branch-1 branch-2 branch-3
```

**Comment by divineunited** [comment]
Do you want to make the change? 😆  -- not sure when I will have time to do it. But I can do it at some point.

**Comment by thwonghin** [comment]
Yeah I can help on this (and others) at some point if they are not already done.
